tests/foreman/ui_airgun/test_domain.py

```diff
@@ -31,7 +31,7 @@
 
 @pytest.fixture
 def valid_domain_name():
-    return valid_domain_names(interface=None)[0]
+    return list(valid_domain_names(interface='ui')['argvalues'])[0]
```

Comment:
a little bit complex structure. any reason for it?

Reply:
With `interface=None` it will generate SMP utf8 chars which aren't supported in chrome, so from time to time tests will fail.
`interface='ui'` knows we use pytest's parametrize so it converts a list to dict acknowledgeable by parametrize, that's why we have to do some extra steps to unpack it.